src/KernelCandidate.cc

```diff
@@ -8,7 +8,7 @@
  *
  * @ingroup ip_diffim
  */
-
+#include <stdexcept>
```

Comment:
Is this the LSST exception module?

Reply:
Nope, `<>` includes are from the standard library. It was necessary for `std::logic_error`.